### PR TITLE
Fix icons with npm install

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: server/lib/market.coffee [hunk 1]
```diff
@@ -46,6 +27,10 @@ download = module.exports.download = (callback) ->
             version = oldMarket.version
             commit = oldMarket.commit
 
+    # Turn this two line one to experiment market locally
```
Comment (nono) [inline server/lib/market.coffee]: s/one/on/ ?

File: server/lib/icon.coffee [hunk 2]
```diff
@@ -6,6 +6,7 @@ log = require('printit')
 module.exports = icons = {}
 market = require './market'
 
+APPS_FOLDER = '/' + path.join 'usr', 'local', 'cozy', 'apps'
```
Comment (nono) [inline server/lib/icon.coffee]: Why not just `'/usr/local/cozy/apps'`?

Comment (frankrousseau) [inline server/lib/icon.coffee]: The right way to do is using path. But what is weird is that '/' at the beginning of the string construction.

Comment (nono) [inline server/lib/icon.coffee]: It's the right way if you want the path to be portable to OS other than UNIX. And `/usr/local` is already a UNIX thing, so you won't be more portable by using `path.join`.

Comment (frankrousseau) [inline server/lib/icon.coffee]: I understand your point but using path is a kind of good practice. It's like a guideline.

**Pull request merged**